app/src/main/java/com/example/tasky/feature_agenda/presentation/AgendaViewModel.kt

```diff
@@ -212,10 +212,10 @@ class AgendaViewModel @Inject constructor(
     fun formatTimeOnAgendaCard(fromDate: Long, toDate: Long? = null): String {
         println("fromDate $fromDate")
 
-        val startDate = DateTimeHelper.formatEpochToDateString(fromDate)
+        val startDate = fromDate.convertMillisToMMMdHHmm()
```

Comment:
Instead of having a function that directly contains the Long timestamp to the formatted date/time, I'd recommend working with `ZonedDateTime` objects in your domain model instead (which you can then format in the UI). The advantage is that this makes it a lot easier to deal with times and dates, e.g. to add a few days, get time between two dates, etc. (whatever you may need, but it's just much easier to work with ZonedDateTime than with a Long timestamp)